modules/phase_field/doc/content/documentation/systems/Modules/PhaseField/GrandPotential/index.md

```diff
@@ -0,0 +1,10 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
```

Comment:
@aeslaughter, what is the purpose of these empty index files? I suppose they are useful if there are multiple objects below a new syntax hierarchy. Is there any link from this page to the actual documentation meat? (I guess I can look myself..)

Reply:
Ok, there is an automatically generated link, so this is fine.